lib/netsuite/records/item_vendor_list.rb

```diff
@@ -0,0 +1,13 @@
+module NetSuite
+  module Records
+    class ItemVendorList
```

Comment:
@rudyyazdi Is there a reason the `Support::Sublist` base class wouldn't work for you here? I'd prefer to use that base class for all sublists if possible.